CHANGES.md

```diff
@@ -1,5 +1,152 @@
 # Changes
 
+## stable-2.12.0
+
+This release introduces route-based policy to Linkerd, allowing users to define
+and enforce authorization policies based on HTTP routes in a fully zero-trust
+way. These policies are built on Linkerd's strong workload identities, secured
+by mutual TLS, and configured using types from the Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/).
+
+The 2.12 release also introduces access logging (an optional feature that
+enables Linkerd to produce Apache-style request logs), optional support for
```

Comment:
access logs can be apache-style or JSON, fwiw.

Reply:
> The 2.12 release also introduces optional request logging ("access logging"
after its name in webservers), optional support for `iptables-nft`, ...

?

Reply:
Went with that for now.